src/game_state/Player.cpp

```diff
@@ -5,7 +5,7 @@
 
 // static
 const float Player::PLATFORM_SPACING = 0.01f;
-const glm::vec3 Player::INITIAL_POSITION(-3, 3, -5);
+const glm::vec3 Player::INITIAL_POSITION(-5.5, 3, -5);
```

Comment:
yep. Happens on both music->stop() and music->setPlayingOffset(). I'll create an issue for it but idk how much we can do since its an internal thing and is only happening on my mac. Couldn't find any recent similar issues online :disappointed: